### PR TITLE
[flash_ctrl] Forgot to update design stage to D2

### PR DESCRIPTION
**Pull request by tjaychen**
Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

**Comment by tjaychen** [comment]
could someone help me merge this once approved?

**Review by eunchan — APPROVED** [review]

**Comment by eunchan** [comment]
Of course I can :)

**Pull request merged**
